pkg/serviceactivity/serviceactivity.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2022 Outreach Corporation. All Rights Reserved.
+
+// Description: This file defines the ServiceActivity type
+// to be implemented by services.
+
+// Package serviceactivity provides service activities for services
+// to implement and a framework to run them.
+package serviceactivity
```

Comment:
```suggestion
package service
```

Reply:
but it's not a generic service package 😉

Reply:
could it ever be? i still think my naming suggestions are better regardless